package-lock.json

```diff
@@ -4,6 +4,57 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@sinonjs/commons": {
```

Comment:
are these all for the new tests? seems like a lot 🤷🏻‍♀️

Reply:
Yup, all from adding Sinon 😕 The only alternative I know of is [Jest mocks](https://jestjs.io/docs/mock-functions), but we already have Mocha and Chai set up in this repo.

I'm happy to swap all three out for Jest later if that's what we want for this repo!